eth/catalyst/api_test.go

```diff
@@ -184,6 +184,31 @@ func checkLogEvents(t *testing.T, logsCh <-chan []*types.Log, rmLogsCh <-chan co
 	}
 }
 
+func TestInvalidPayloadTimestamp(t *testing.T) {
+	genesis, preMergeBlocks := generatePreMergeChain(10)
+	n, ethservice := startEthService(t, genesis, preMergeBlocks)
+	ethservice.Merger().ReachTTD()
+	defer n.Close()
+	var (
+		api    = NewConsensusAPI(ethservice, nil)
+		parent = ethservice.BlockChain().CurrentBlock()
+	)
+	params := PayloadAttributesV1{
+		Timestamp:             parent.Time() - 1,
```

Comment:
Repeating the parent timestamp should be enough for it to be invalid. The yellow paper defines this in (50): `H_s > P(H)_{H_s}`

Reply:
You're right, will fix